### PR TITLE
Check if .zprofile exists

### PR DESCRIPTION
**Pull request by xobust**
# Pull Request Check List

Resolves: #507

- [ ] Added **tests** for changed code.
- [ ] Updated **documentation** for changed code.

I keep getting email notifications from users running in to #507 so I made a quick fix for it.
Let me know if it needs improvements.

### CODE REVIEW

File: get-poetry.py [hunk 1]
```diff
@@ -903,7 +903,12 @@ def get_unix_profiles(self):
 
         if "zsh" in SHELL:
             zdotdir = os.getenv("ZDOTDIR", HOME)
-            profiles.append(os.path.join(zdotdir, ".zprofile"))
+            zprofile = os.path.join(zdotdir, ".zprofile")
+            zshrc = os.path.join(zdotdir, ".zshrc")
+            if os.path.isfile(zprofile):
+                profiles.append(zprofile)
+            else:
+                profiles.append(zshrc)
```
Comment (abn) [inline get-poetry.py:911]: Is `.zshrc` more common than `.zprofile`?

Comment (xobust) [reply get-poetry.py:911]: For me the zsh installation only created a `.zshrc` others have ran into this issue as well (#507)

Comment (HHogg) [reply get-poetry.py:911]: Just another voice: I've only ever had/used a .zshrc file, so I just had this issue trying to install.

Comment (JayWelsh) [reply get-poetry.py:911]: Also just want to mention that I have only ever had a `.zshrc` and I think it is worth having the measure in place to use `.zshrc` if `.zprofile` doesn't exist

Comment (xobust) [reply get-poetry.py:911]: @abn Any update?

**Comment by jleclanche** [comment]
Can be closed per #898 @finswimmer

**Pull request closed without merging**